src/router.ts

```diff
@@ -1,12 +1,34 @@
 import { createMemoryHistory, createRouter } from "vue-router";
 
-const MainPage = import('@/pages/MainPage.vue');
+const MainPage = import('@/components/layouts/BaseLayout.vue');
```

Comment:
Хорошая практика - импортировать страницы динамически (загрузка в момент выполнения, а не компиляции)
```suggestion
const MainPage =() => import('@/components/layouts/BaseLayout.vue');
```